salem/tests/test_gis.py

```diff
@@ -1035,8 +1035,8 @@ def test_pyproj_trafo(self):
         assert_allclose(xx, x)
         assert_allclose(yy, y)
 
-        xx, yy = pyproj.transform(gis.check_crs('epsg:26915'),
```

Comment:
Here as well

Reply:
Same here as well, the identical check with gis.transform_proj is directly following it.